karma.conf.js

```diff
@@ -67,7 +70,10 @@ module.exports = function(karma) {
 		rollupPreprocessor: {
 			plugins: [
 				resolve(),
-				commonjs()
+				commonjs(),
+				// The two plugins below are required to load the BasicChartWebWorker.
+				stylesheet({extract: true}),
```

Comment:
The windows failure and coverage drop is probably related to this / these changes. Its running only one test.

> TOTAL: 1 SUCCESS
> TOTAL: 1 SUCCESS

Reply:
I wonder if https://github.com/chartjs/Chart.js/pull/7104 will make the tests pass since it'd make this line unnecessary

Reply:
@davidswinegar we removed all the css stylesheet stuff in #7104, which was just merged. I'm afraid that PR might have made the rebase a little trickier, but hopefully it will save you time in trying to figure out the situation with the tests

Reply:
I'm wondering if `webWorkerLoader` and `resolve`/`commonjs` are interfering with each other